pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -394,7 +394,7 @@ func checkCrcDnsmasqConfigFile() (bool, error) {
 
 func fixCrcDnsmasqConfigFile() (bool, error) {
 	logging.Debug("Fixing dnsmasq configuration")
-	cmd := exec.Command("sudo", "tee", crcDnsmasqConfigPath)
+	cmd := exec.Command("sudo", "tee", crcDnsmasqConfigPath) // #nosec G204
```

Comment:
why not use -exclude G204 from the gosec invocation?

Reply:
@cfergeau we can but then what if in future we start taking some arg from user and pass it to command, It is there just to be in safer side for audit purpose.